packages/back-end/src/routers/saved-queries/saved-queries.controller.ts

```diff
@@ -361,20 +431,32 @@ export async function postGenerateSQL(
       const columnsDescription = value.columns
         .map((column) => `${column.columnName} (${column.dataType})`)
         .join(", ");
-      return `Database: ${value.databaseName}, Table: ${value.tableName}, Schema: ${value.tableSchema}, Columns: [${columnsDescription}]`;
+      return `Database: ${value.databaseName}, Table: ${value.tableName}, Schema name: ${value.tableSchema}, Columns: [${columnsDescription}]`;
     }
   ).join("\n");
 
-  const instructions =
+  let instructions =
     "You are a data analyst and SQL expert.\n" +
     "Generate a SQL query to answer the provided question inputted. The resultant query should be ONLY based on the provided context of tables and structure, and return only valid SQL that is executable on the specified data source." +
+    "\n\nThe database is a " +
+    datasource.type +
```

Comment:
Will need to special case `growthbook_clickhouse` to just render as `clickhouse` here.